src/main/java/bacnet/Utils.java

```diff
@@ -772,32 +772,35 @@ public static EventTransitionBits jsonArrayToAckRequired(JsonArray jarr) {
 		String valuesStr = Arrays.toString(enumData.getEnumConstants());
 		return valuesStr.substring(1, valuesStr.length() - 1).replace(" ", "").split(",");
 	}
-	
-	public  static String[] enumeratedObjectTypeNames(){
+
+	public static String[] enumeratedObjectTypeNames() {
 		String valuesStr = Arrays.toString(ObjectType.ALL);
 		return valuesStr.substring(1, valuesStr.length() - 1).replace(" ", "").split(",");
 	}
-	
-	public static  ObjectType parseObjectType(String typeString) {
 
-        for (ObjectType type: ObjectType.ALL){
-        	if (type.toString().replace(" ", "").equals(typeString.replace(" ", ""))){
-        		return type;
-        	}
-        }	
+	public static ObjectType parseObjectType(String typeString) {
+
+		for (ObjectType type : ObjectType.ALL) {
+			if (type.toString().replace(" ", "").equals(typeString.replace(" ", ""))) {
```

Comment:
Is that meant to remove all spaces, or just trim outside ones? If it's for the latter, use String.trim() instead.